cloudevents/http/http_methods.py

```diff
@@ -16,37 +16,13 @@
 
 from deprecation import deprecated
 
-from cloudevents.conversion import from_http as _abstract_from_http
-from cloudevents.conversion import to_binary, to_structured
+# backwards compatibility
+from cloudevents.http.conversion import from_http  # noqa
+from cloudevents.http.conversion import to_binary, to_structured
```

Comment:
I believe we should use the `deprecation` for these backward-compatibility exports.

Please consider marking all "backward-compatible" APIs as deprecated so that users know they should migrate to the new onces.

Reply:
I won't duplicate this comment, but this applies to a lot of different places across the codebase.